setup.cfg

```diff
@@ -17,8 +17,8 @@ classifiers =
     Topic :: System :: Monitoring
     Programming Language :: Python :: Implementation :: PyPy
     Programming Language :: Python :: Implementation :: CPython
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
+    ; Programming Language :: Python :: 2
+    ; Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3.6
```

Comment:
Remaining cleanup to remove Python 3.6 support and add 3.11 will be completed in a separate PR.